graylog2-server/src/main/java/org/graylog2/alerts/AbstractAlertCondition.java

```diff
@@ -191,7 +191,7 @@ public DateTime getTriggeredAt() {
 
     public static class NegativeCheckResult extends CheckResult {
         public NegativeCheckResult(AlertCondition alertCondition) {
-            super(false, alertCondition, null, null, null);
+            super(false, null, null, null, null);
```

Comment:
This was changed on purpose. If the CheckResult is negative, there is no triggered alert condition. Removing the parameter makes sense though.